docker-compose.yaml

```diff
@@ -12,19 +12,17 @@ services:
       - /usr/src/app
       - /usr/src/app/node_modules
     ports:
-      - 8000:8000
+      - '8000:8000'
     # command: npm run dev
     depends_on:
       - mongo
     environment:
       PORT: ${PORT}
-      MONGO_CONNECTION_STRING: mongodb://mongo:27017/mmict-test
```

Comment:
Why remove this?

Reply:
At first I thought it might be a security vulnerability, but now when I look at it since it's an internal connection to a docker container it should be fine. I'll make sure to include it in this pull request